lib/js/components/SectionHeader/SectionHeader.vue

```diff
@@ -118,10 +124,15 @@ $icons-and-slot-min-height-l: 50px;
 			margin-left: $space-xxs;
 		}
 
-		#{$root}__slot,
 		#{$root}__iconWrapper {
 			min-height: $icons-and-slot-min-height-l;
 		}
+
+		#{$root}__slot {
+			@media #{breakpoint-s()} {
+				min-height: $icons-and-slot-min-height-l;
```

Comment:
IMO, `height` should be set on `.sectionHeader` and rest should be centered using flex. Right now, you have to set height in multiple places

Reply:
I think it's connected with above comment, where we need to center icons in the center when text has 1 line, but stick to the top when text has more than 1 line.